stable/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 1.31.2
+version: 1.31.3
```

Comment:
could you please update the chart version.

```suggestion
version: 1.31.4
```

Reply:
Updated